src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -16,7 +18,7 @@
  */
 public class AddCommand extends Command {
 
-    public static final String COMMAND_WORD = "add";
+    public static final List<String> COMMAND_WORD = List.of(new String[]{"add", "a"});
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
```

Comment:
Does this line still produce the right output? Could either stick to the original text, or if anyone has ideas could also somehow introduce the new shortcut to the user

Reply:
Yeah this works fine, so after this I will allow users to define which shortcut they want